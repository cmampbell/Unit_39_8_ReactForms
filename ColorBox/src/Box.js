import React from "react";
import "./Box.css"

const Box = ({boxStyle, deleteBox}) => (
    <div style={boxStyle}>
        <button onClick={() => deleteBox(boxStyle)}>X</button>
    </div>
)

export default Box;