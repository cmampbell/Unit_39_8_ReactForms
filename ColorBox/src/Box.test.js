import React from 'react';
import { render } from "@testing-library/react";
import Box from './Box';


it('renders without crashing', function() {
    render(<Box boxStyle={{height: 10, width: 10, color: '#000000'}}/>);
});

it('matches snapshot', function() {
    const {asFragment} = render(<Box boxStyle={{height: 10, width: 10, color: '#000000'}}/>);
    expect(asFragment()).toMatchSnapshot();
});