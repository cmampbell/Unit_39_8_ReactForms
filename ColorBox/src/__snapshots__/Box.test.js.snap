// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    style="height: 10px; width: 10px; color: rgb(0, 0, 0);"
  >
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
